discord/code_processing.py

```diff
@@ -41,10 +41,10 @@ def _processing(lang, code):
     }
 
     regexes = {
-        "cpp": "(?:int|void) main(?:\s*)\(.*\)",
-        "c": "(?:int|void) main(?:\s*)\(.*\)",
-        "go": "func main(?:\s*)\(.*\)",
-        "rs": "fn main(?:\s*)\(.*\)"
+        "cpp": "(?:int|void) main(?:\\s*)\\(.*\\)",
+        "c": "(?:int|void) main(?:\\s*)\\(.*\\)",
+        "go": "func main(?:\\s*)\\(.*\\)",
+        "rs": "fn main(?:\\s*)\\(.*\\)"
```

Comment:
These do not correctly scan for a `main()` function, but the original does.

Please consider reverting or defend the changes.

Reply:
Interesting — in my tests, these still scan correctly, the same as the original. I changed it because my editor was displaying warnings here, and I just noticed that these are all invalid escape sequences, so I figured I'd clean up the file.

Anyway, this is secondary to the pull request, so I'll go ahead and revert the changes.

Reply:
I don't know why it would be worried about invalid escape sequences. I want to scan for a literal `(` character (and not be interpreted as a group), so I don't know why we would need to escape the escape on the `(`. Either way, LGTM